README.md

```diff
@@ -46,7 +69,77 @@ npm install
 
 ## Usage
 
-TBD
+This client assumes:
+
+* a remote Wallet Attached Storage server is available
+* the app is able to create and store a public-private key pair, see the
+  [App Identity and Key Management](#app-identity-and-key-management) section
+  for more details.
+
+Example key pair generation (for an in-browser SPA), for persistence in local
+storage.
+
+```ts
+import { Ed25519Signer } from '@did.coop/did-key-ed25519'
+
+const appDidSigner = await Ed25519Signer.generate()
+```
+
+Example storing and loading:
+
+```ts
+// Serialize to JSON for storage
+const exportedKeyPair = appDidSigner.toJSON()
+localStorage.setItem('app-DID', JSON.stringify(exportedKeyPair))
+
+// Load from storage and turn back into a DID Signer
+const loadedKeyPair = localStorage.getItem('app-DID')
+const appDidSigner = Ed25519Signer.fromJSON(loadedKeyPair)
+```
+
+Create a Wallet Attached Storage Client:
+
+```ts
+import { WasClient } from '@did-coop/wallet-attached-storage'
+
+const config = {
+  was: {
+    url: 'https://data.pub'
+  }
+}
+
+const storage = new WasClient({ url: config.was.url, signer: appDidSigner })
+```
+
+Provision (create and set up) a Data Space (this performs a [Create Space HTTP
+API operation](https://did-coop.github.io/wallet-attached-storage-spec/#create-space-operation)):
+
+```ts
+let space
+try {
+  space = await storage.space('demo-space')
```

Comment:
This is similar to the fetch-client API, but slightly different. e.g. that one has `pub.space()` but returns synchronously

Reply:
user will have zcap to space, which will contain a space id (urn:uuid), and they can create a space from that like `storage.space(uuid)`